client/app/bundles/buttons/button.scss

```diff
@@ -0,0 +1,79 @@
+@import "base.scss";
+
+
+.buttonFilledM{
+         font-size: 18px;
+         color: #6D0839;
+         font-weight: bold;
+         background-color: #FFFFFF;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         padding-left: 85px;
+         padding-right: 85px;
+         border-radius: 5px;
+         border-color: #FFFFFF;
+         border-width: 0px;
+         cursor: pointer;
+ }
+
+.buttonGhostM{
+         font-size: 18px;
+         color: #FFFFFF;
+         font-weight : bold;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         padding-left: 48px;
+         padding-right: 47px;
```

Comment:
Let's make sure the horizontal padding is even here. I think it will also make sense to reduce the horizontal padding to around 24px and adding a `min-width` so that it scales nicely!